docs/get-started/json-rpc-commands.mdx

```diff
@@ -0,0 +1,879 @@
+
+Expect that all properties are optional. 
+
+<b> Object </b>  - The transaction call object
+
+*  <b>  from: DATA, 20 Bytes </b>  - The address the transaction is sent from.
+*  <b>  to: DATA, 20 Bytes </b>  - The address the transaction is directed to.
+*  <b>  gas: QUANTITY </b>  - Integer of the gas provided for the transaction execution. eth_call consumes zero gas, but this parameter may be needed by some executions.
+*  <b>  gasPrice: QUANTITY </b>  - Integer of the gasPrice used for each paid gas
+*  <b>  value: QUANTITY </b>  - Integer of the value sent with this transaction
+*  <b>  data: DATA </b>  - Hash of the method signature and encoded parameters. For details see Ethereum Contract ABI in the Solidity documentation
+*  <b>  QUANTITY|TAG </b>  - integer block number, or the string "latest", see the default block paramete
+
+<h4><i>Returns:</i></h4>
+
+*  <b>  QUANTITY </b> - the amount of gas used.
+
+<h4><i>Example:</i></h4>
+
+````bash
+curl  https://rpc.poa.psdk.io:8545 -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"eth_estimateGas","params":[{see above}],"id":1}'
+````
+
+### eth_newFilter
+
+Creates a filter object, based on filter options, to notify when the state changes (logs).
+To check if the state has changed, call eth_getFilterChanges.
+
+---
+
+<h4><i>Parameters:</i></h4>
+<b> Object </b> - The filter options:
+
+*  <b>  fromBlock: QUANTITY|TAG </b> - (optional, default: "latest") Integer block number, or "latest" for the last mined block 
+*  <b>  toBlock: QUANTITY|TAG </b> - (optional, default: "latest") Integer block number, or "latest" for the last mined block 
+*  <b>  address: DATA|Array, 20 Bytes </b> - (optional) Contract address or a list of addresses from which logs should originate.
+*  <b>  topics: Array of DATA </b> - (optional) Array of 32 Bytes DATA topics. Topics are order-dependent. Each topic can also be an array of DATA with “or” options.
+
+<h4><i>Returns:</i></h4>
+
+*  <b> QUANTITY </b> - A filter id.
+
+<h4><i>Example:</i></h4>
+
+````bash
+curl  https://rpc.poa.psdk.io:8545 -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"eth_newFilter","params":[{"topics":["0x12341234"]}],"id":1}'
+````
+
+### eth_newBlockFilter
+
+Creates a filter in the node, to notify when a new block arrives.
+To check if the state has changed, call eth_getFilterChanges.
+
+---
+
+<h4><i>Parameters:</i></h4>
+
+None
+
+<h4><i>Returns:</i></h4>
+
+1. QUANTITY - A filter id.
+
+<h4><i>Example:</i></h4>
+
+Run command and see live results from our testnet.
+
+````bash
+curl  https://rpc.poa.psdk.io:8545 -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"eth_newBlockFilter","params":[],"id":1}'
+````
+
+<JsonRpcTerminal method="eth_newBlockFilter" params={[]} network="https://rpc.poa.psdk.io:8545"/>
+
+### eth_getFilterChanges
+
+Polling method for a filter, which returns an array of logs which occurred since last poll.
```

Comment:
Change to "....an array of logs that occurred since the last poll"

Reply:
Fixed! 
 [b1173b4](https://github.com/0xPolygon/polygon-sdk-docs/pull/41/commits/b1173b4db2a92b5f3c153e1906ad8ac9e82900bf)